packages/template/typescript-webpack/src/TypeScriptWebpackTemplate.ts

```diff
@@ -26,6 +26,9 @@ class TypeScriptWebpackTemplate extends BaseTemplate {
                 html: './src/index.html',
                 js: './src/renderer.ts',
                 name: 'main_window',
+                preload: {
+                  js: 'preload.js',
```

Comment:
Does the src path need provided too?
```suggestion
                  js: './src/preload.ts',
```

Reply:
It does. I just needed to fix that. I will commit your changes